08-rails/indirect-relations.md

```diff
@@ -0,0 +1,127 @@
+# Indirect Model Relations
+
+## Learning Goals
+
+By the end of this lesson, students should be able to...
+
+- Identify when an indirect model relation would be useful
+- Set up an indirect relation using `has_many through:`
+- Distinguish between indirect relations and join tables
+
+## Introduction
+
+This lesson builds directly on the lesson on [many to many relationships](many-to-many-relationships.md), so make sure you've got that under your belt before getting started here.
```

Comment:
To me, this implies you must fully understand that lesson before starting this one. With the timing of things, I think it’d be best to mention that it builds, but that they don’t have to have 100% comprehension... especially if we aren’t planning to give them time to practice in between.